app/controllers/items_controller.rb

```diff
@@ -0,0 +1,60 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: [:show, :edit, :update, :destroy]
```

Comment:
I like using this before_action, and this definitely works. I would just be sure to be aware that if you had some nested resources using any actions in this controller, and had a param come in that wasn't called `:id`, and maybe `:item_id`, then you'd have to take that into account. But I like your creativity here.